qemu/tests/cfg/virtio_fs_share_data.cfg

```diff
@@ -296,6 +296,13 @@
                     generic_blacklist = 'generic/003\ngeneric/120\ngeneric/426\ngeneric/467\ngeneric/477\ngeneric/551\n'
                     with_xfstest..with_cache.always:
                         generic_blacklist += 'generic/451\n'
+                    # These tests not support mmap with cache=none, id2229115
```

Comment:
We don't recommend adding the internal information here since some upstream users can not access it for details, so we could summary it here.